### ISSUE TITLE
Plot statistics of the vcf files

### ISSUE THREAD

**Issue by IgnasiLucas** [IgnasiLucas/mosquito#15]
The new version of bcftools (not the one installed, but freely available: http://www.htslib.org) includes a script that authomatically generates some plots that can be very useful to assess the quality of the data. Take a look at the manual.

**Comment by mgmorant** [comment]
I downloaded samtools and bcftools but im using it in the terminal and is not reconized(the new version). I have them in Escritori, Have I got to do something to make they work? Also, I searched how to make it and I found that the option is '-p'. Do I have to use it only when I had used bcftools or I can use it to any file.vcf?

**Comment by IgnasiLucas** [comment]
I guess you may be just typing the name of the program. Instead, type the whole path to the version that you downloaded. That should make the trick. I suppose you can use bcftools with any vcf file. However, not all vcf files contain the same amount of information. If some fields are missing, some features of bcftools may not work. Give it a try, @mgmorant .

**Comment by mgmorant** [comment]
I've tried and I cant

**Comment by IgnasiLucas** [comment]
It's certainly difficult. This bcftools requires several specific packages to do the plots. I manage to install some of them, and it does produce some plots. There are several steps:
bgzip input.vcf                                      # to compress the vcf file.
tabix input.vcf.gz                                   # to index the compressed vcf file.
~/bin/bcftools stats input.vcf.gz > z1      # to generate statistics.
mkdir plots                                             # to create a target folder
~/bin/plot-vcfstats -p plots/ z1                # to plot the statistics.

Feel free to start a new analysis in a new folder. You can plot the statistics for the two vcf files together (the one generated with mplileup and the one generated with freebayes.

BTW, I installed the new bcftools in your home directory. So, if you type '~/bin/bcftools' you access the new one, while if you type  just 'bcftools', the old one is exectued by default.

**Comment by mgmorant** [comment]
Ignasi Im havinig some problems with github since I merged with you. I have to remove many things that are from 2016-07-06 and 06b and i cant because im executing some files and also, because if i have to run everything again it will be a lot of time.
For that reason Im writing the new files on github directly but there is a problem: you only can create files and not directories. In thermomix I've done a new directory 2016-08-05, but i cant push it so I wrote the file as README.sh in results.
Sorry
